### ISSUE TITLE
It seems that you need to upgrade to the latest React dependency

### ISSUE THREAD

**Issue by MonkeyJumps** [jchristman/react-context-menus#2]
Can you take a look at the following warning messages. Would  updating the react dependency resolve some of these warnings? I forked your repo but i could not get the build scripts to run on my windows machine. I am using MINGW64

![image](https://user-images.githubusercontent.com/812636/30509898-79fbf786-9a6f-11e7-93fc-1ec75ed60a8e.png)

**Comment by jchristman** [comment]
Interesting. I’ll take a look at that on Monday. Thanks for the report!

**Comment by MonkeyJumps** [comment]
Hi @jchristman any luck? the context menus work fine. However, these ugly warnings are appearing.

**Comment by jchristman** [comment]
Oops totally forgot. Will look at it tonight!

**Comment by jchristman** [comment]
You are not forgotten. I'm getting there. But golly updating dependencies is a pain. Just since the last time I looked at this, a whole crapton of webpack configs have to change. I'll prolly finish up tomorrow night.

**Comment by jchristman** [comment]
Alright - After debugging a bit, I have come to the conclusion that https://www.npmjs.com/package/react-menus is causing all of those errors (the underlying library I was wrapping around). It seems, however, that their github page is gone, which is just fantastic. I will consider pulling that source code into this repository so that I can clean up those warning, but my time is reasonably limited.

In the meantime, you may just have to live with those warnings. Sorry!

**Comment by MonkeyJumps** [comment]
Yes I noticed their page was gone. I understand, I am still moving forward with your component, because it is working well despite the warning messages. Thank you for any help you can provide. Please keep me posted.

**Comment by MonkeyJumps** [comment]
any progress on upgrading this package? I would like update to React 16 but have your component as a dependency

**Comment by jchristman** [comment]
All of my free time right now is sucked up by a certification class I’m in until late Feb. I’d accept a pull request and publish it to npm.

**Comment by MonkeyJumps** [comment]
I cloned your repo and tried to make changes but could not run the build scripts successfully.  However, I was able to manually update the code inside my /node_modules directory. As a result, I had to use the create-react-class for this to work. 

Can i provide a zip file with this changes for your review and possible acceptance?

**Comment by jchristman** [comment]
Sure thing. It does not surprise me that the build scripts are in not great shape, this now being an old package that I don’t have tons of time to maintain. Happy to push updates though

**Comment by MonkeyJumps** [comment]
here are the files. I just addded the create-react-class to the package.json of each project and then imported it where ever React.createClass is used.

https://www.dropbox.com/s/os5hmgrbbf3lg7w/react-context-menus.zip?dl=0
https://www.dropbox.com/s/jhoj9d7a6efdbzh/react-menus.zip?dl=0
